README.md

```diff
@@ -80,7 +80,7 @@ sudo debuginfo-install kernel-${UNAME%.*}
 sudo yum install gcc-plugin-devel
 
 # optional, but highly recommended
-sudo yum install https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/c/ccache-3.3.4-1.el7.x86_64.rpm
+sudo yum install https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/c/ccache-3.7.6-1.el7.x86_64.rpm
```

Comment:
I have a kpatch setup script that just installs the epel, ccache and then removes epel to avoid this issue.   Feel free to steal it if you are tired of updating these.  (It should probably check to see if epel was installed originally, but I only use the script on test boxes.)
``` shell
# ccache from EPEL
yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${RHEL}.noarch.rpm
yum install -y ccache
ccache --max-size=5G
yum erase -y epel-release
```

Reply:
That is a great suggestion, I'll apply that.